.template-env

```diff
@@ -4,6 +4,7 @@ NB_GRAPH_PASSWORD=DBPASSWORD  # REPLACE DBPASSWORD WITH YOUR GRAPH DATABASE PASS
 NB_GRAPH_DB=test_data/query
 NB_RETURN_AGG=true
 NB_API_TAG=latest
+NB_API_ALLOWED_ORIGINS="https://localhost:3000 http://localhost:3000"
```

Comment:
If I understand correctly, you want to show an example of two origins here? That makes sense to me. This is a minor comment, but I think it'd be better to use something like 
```suggestion
NB_API_ALLOWED_ORIGINS="https://localhost:3000 https://myOtherOrigin"
```
instead for the second option to highlight that we aren't advising people to configure their API for both HTTP and HTTPS. If they configure HTTPS, they likely also have NGINX setup, so they may not even need to configure CORS at all. For the folks who just quickly follow the defaults, best to keep it simple. Maybe a comment helps

Reply:
Okay, that's a fair point regarding http/https.

I wanted to provide a sensible template value that the user would not have to edit if following our recommended setup with a locally deployed query tool. In this case since I wasn't sure if they would have HTTPS configured I listed both to cover all the bases.

Instead of `https://myOtherOrigin` as the second origin (which a user can't use as-is), how about `https://127.0.0.1:3000` to cover both ways of accessing the query tool on localhost? Alternatively, we could have `https://query.neurobagel.org` as the second origin, but I'm not sure if users would want to make a local API accessible by our tool by default.

Reply:
I see. OK, then how about this: keep your original version, just a comment above to say something like:
`# template for adding multiple origins, please edit this line for your own setup before deploying`?